tests/System.Text.Primitives.Tests/Parsing/PrimitiveParserSBytePerfTests.cs

```diff
@@ -0,0 +1,270 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+using Microsoft.Xunit.Performance;
+
+namespace System.Text.Primitives.Tests
+{
+    public partial class PrimitiveParserSBytePerfTests
+    {
+        private const int InnerCount = 10000;
+
+        private static readonly string[] s_SByteTextArray = new string[20]
+        {
+            "145",
+            "2",
+            "112",
+            "-112",
+            "114",
+            "-21",
+            "-2",
+            "114",
+            "-114",
+            "-124",
+            "117",
+            "-117",
+            "-14",
+            "-2",
+            "2",
+            "14",
+            "74",
+            "21",
+            "83",
+            "-127"
+        };
+
+        [Benchmark(InnerIterationCount = InnerCount)]
+        [InlineData("107")] // standard parse
+        [InlineData("127")] // max value
+        [InlineData("0")]
+        [InlineData("-128")] // min value
+        private static void PrimitiveParserByteSpanToSByte(string text)
```

Comment:
We don't need to test this overload, especially since we will likely remove this API.